pkgs/by-name/wh/whatsie/package.nix

```diff
@@ -2,12 +2,13 @@
   fetchFromGitHub,
   lib,
   stdenv,
+  makeDesktopItem,
   libX11,
   libxcb,
   qt5,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
When you have `finalAttrs` the `rec` keyword can (and should) be removed. Access the attribute you want with `finalAttrs.myAttr` instead of `myAttrs`.

Reply:
Or in this case it can simply be dropped, as the expression relies on the `$desktopItem` environment variable at build time instead of the Nix attribute at eval time.